### ISSUE TITLE
Tax-Brain dependencies

### ISSUE THREAD

**Issue by jdebacker** [PSLmodels/Tax-Brain#121]
After building the `taxbrxin-dev` environment and trying to run the pytest suite, I got a warning that I needed additional packages:

```
RuntimeError: Neither firefox and geckodriver nor a variant of chromium browser and chromedriver are available on system PATH. You can install the former with 'conda install -c conda-forge firefox geckodriver'.
```

Shall I add `firefox` and `geckodriver` to the `environment.yml`? 

cc @Peter-Metz @andersonfrailey

**Comment by Peter-Metz** [comment]
Interesting, I'm not getting that warning but Google tells me this could be an issue with the `selenium` dependency.

@andersonfrailey do you know what Tax-Brain uses Selenium for?

**Comment by andersonfrailey** [comment]
@Peter-Metz @jdebacker, selenium is used by Bokeh to create the png outputs in Tax-Brain's automated reports. Here's the section of the [Bokeh docs](https://docs.bokeh.org/en/latest/docs/user_guide/export.html) where they mention it.

**Comment by andersonfrailey** [comment]
This issue was resolved with PR #123, closing.
